data/trie/branchNode.go

```diff
@@ -779,6 +779,30 @@ func (bn *branchNode) getAllHashes(db data.DBWriteCacher) ([][]byte, error) {
 	return hashes, nil
 }
 
+func (bn *branchNode) getNumNodes() data.NumNodesDTO {
+	arg := data.NumNodesDTO{
+		Total:    1,
+		Branches: 1,
+	}
+
+	for _, n := range bn.children {
+		if n != nil {
```

Comment:
wondering why isInterfaceNil is not exported. It would have worked with check.IfNil. @BeniaminDrasovean is there a reason to keep isInterfaceNil unexported?

Reply:
done